l10n_br_base/models/res_company.py

```diff
@@ -85,79 +85,86 @@ def _inverse_suframa(self):
-        inverse='_inverse_district',
-        size=32)
+        string="District",
+        compute="_compute_l10n_br_data",
+        inverse="_inverse_district",
+        size=32,
+    )
 
     city_id = fields.Many2one(
-        string='City of Address',
-        comodel_name='res.city',
+        string="City of Address",
+        comodel_name="res.city",
         domain="[('state_id', '=', state_id)]",
-        compute='_compute_l10n_br_data',
-        inverse='_inverse_city_id')
+        compute="_compute_l10n_br_data",
+        inverse="_inverse_city_id",
+    )
 
-    country_id = fields.Many2one(
-        default=lambda self: self.env.ref('base.br'))
+    country_id = fields.Many2one(default=lambda self: self.env.ref("base.br"))
 
     cnpj_cpf = fields.Char(
-        string='CNPJ',
-        compute='_compute_l10n_br_data',
-        inverse='_inverse_cnpj_cpf',
-        size=18)
+        string="CNPJ",
+        compute="_compute_l10n_br_data",
+        inverse="_inverse_cnpj_cpf",
+        size=18,
+    )
 
     inscr_est = fields.Char(
-        string='State Tax Number',
-        compute='_compute_l10n_br_data',
-        inverse='_inverse_inscr_est',
-        size=16)
+        string="State Tax Number",
+        compute="_compute_l10n_br_data",
+        inverse="_inverse_inscr_est",
+        size=16,
+    )
 
     state_tax_number_ids = fields.One2many(
-        string='State Tax Numbers',
-        comodel_name='state.tax.numbers',
-        inverse_name='partner_id',
-        compute='_compute_l10n_br_data',
-        inverse='_inverse_state_tax_number_ids',
-        ondelete='cascade')
+        string="State Tax Numbers",
+        comodel_name="state.tax.numbers",
+        inverse_name="partner_id",
+        compute="_compute_l10n_br_data",
+        inverse="_inverse_state_tax_number_ids",
+        ondelete="cascade",
+    )
 
     inscr_mun = fields.Char(
-        string='Municipal Tax Number',
-        compute='_compute_l10n_br_data',
-        inverse='_inverse_inscr_mun',
-        size=18)
+        string="Municipal Tax Number",
+        compute="_compute_l10n_br_data",
+        inverse="_inverse_inscr_mun",
+        size=18,
+    )
 
     suframa = fields.Char(
-        string='Suframa',
-        compute='_compute_l10n_br_data',
-        inverse='_inverse_suframa',
-        size=18)
+        string="Suframa",
+        compute="_compute_l10n_br_data",
+        inverse="_inverse_suframa",
+        size=18,
+    )
 
     @api.model
-    def _fields_view_get(self, view_id=None, view_type='form',
-                         toolbar=False, submenu=False):
-        res = super(Company, self)._fields_view_get(view_id,
-                                                    view_type,
-                                                    toolbar,
-                                                    submenu)
-        if view_type == 'form':
-            res['arch'] = self._fields_view_get_address(res['arch'])
+    def _fields_view_get(
```

Comment:
É só a formatação que esta mudando?

Reply:
Sim, são padões de formatação que entraram nos checks da OCA da versão 13.0.

Então como a versão 12.0 não esta pronta é ideal que coloquemos estes checks e automações na versão 12 para facilitar a migração para versão 13.0 / 14.0.

Reply:
Sinceramente eu prefiro as aspas simples, mas como isto é um padrão da OCA e o script faz isso automaticamente.

Devemos inclusive colocar um hook para que o script de pré commit sempre seja executado.
Por exemplo: https://medium.com/@rashtay/how-to-run-eslint-using-pre-commit-hook-25984fbce17e